packages/client/src/store/userStore.ts

```diff
@@ -137,6 +140,19 @@ export const useUserStore = create<IUserStore>()(
           }));
         }
       },
+      updateAvatar: async (body) => {
+        try {
+          const { data } = await ApiClient.updateAvatar(body);
+          set(() => ({
+            user: {
+              ...data,
+              isAuth: true,
+            },
```

Comment:
А предыдущее значение `user` не удалится?

```suggestion
          set(({ user }) => ({
            user: {
              ...user,
              ...data,
              isAuth: true,
            },
```

Reply:
Если судить по API, то все поля обязательно должны вернуться.